src/components/views/JobForm.js

```diff
@@ -29,6 +30,39 @@ class JobForm extends React.Component {
     invalid: PropTypes.bool
   }
 
+  renderPaneFooter() {
+    const startButton = (
+      <Button
+        buttonStyle="default mega"
+        id="clickable-cancel"
+        marginBottom0
+        onClick={this.props.handlers.onClose}
+      >
+        <FormattedMessage id="stripes-components.cancel" />
+      </Button>
+    );
+
+    const endButton = (
+      <Button
+        buttonStyle="primary paneHeaderNewButton"
```

Comment:
```suggestion
        buttonStyle="primary mega"
```

Should this be `primary mega`? [Asking based on the ui-users example](https://github.com/folio-org/ui-users/blob/master/src/UserForm.js#L274).